contracts/src/v0.8/l2ep/test/v1_0_0/arbitrum/ArbitrumSequencerUptimeFeed.t.sol

```diff
@@ -246,199 +246,3 @@ contract ArbitrumSequencerUptimeFeed_ProtectReadsOnAggregatorV2V3InterfaceFuncti
     assertEq(answer, 0);
   }
 }
-
-contract ArbitrumSequencerUptimeFeed_GasCosts is ArbitrumSequencerUptimeFeedTest {
```

Comment:
Intentional removal

Reply:
Why did we remove them? Are they not needed / used anymore?

Reply:
These are "bad" tests that only exist in Foundry because they existing in Hardhat. Since Foundry has gas snapshots, these tests have no purpose. This type of test is generally bad as you need to update them for every tiny contract change. In this case, they broke CI because Foundry coverage has different gas usage compared to foundry test